icu4c/source/i18n/nfrule.h

```diff
@@ -66,6 +66,8 @@ class NFRule : public UMemory {
     char16_t getDecimalPoint() const { return decimalPoint; }
 
     int64_t getDivisor() const;
+    
+    bool hasModulusSubstitution() const;
```

Comment:
Please add docs explaining what a "modulus substitution" is and how it relates to flooring the number

Reply:
Done.  Please let me know what you think.